### PR TITLE
Added password derivation

### LINKED ISSUE #3672: Polkadot Accounts does not mention the password derivation
The article about [Polkadot Accounts](https://wiki.polkadot.network/docs/learn-accounts#derivation-paths) mentions only soft and hard derivations. AFAIK, there is another type of derivation which is referred to as "password". 

Was it overlooked, or is this deliberate? 

If it was overlooked, then it should be mentioned. If it was deliberate, there should probably be a notice about it anyway.

### PR DESCRIPTION
**Pull request by filippoweb3**
Added small part about password derivation and how it differs from soft and hard derivation

### CODE REVIEW

**Comment by filippoweb3** [comment]
Run prettify on the branch, no changes.

File: docs/learn/learn-accounts.md [hunk 1]
```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
...
 :::
 
+There is an additional type of derivation called password derivation. On Polkadot you can derive a
+**password key** account using **///** after the mnemonic phrase
+
+```
+'caution juice atom organ advance problem want pledge someone senior holiday very///0'
+```
+
+In this type of derivation, if the seed would leak, accounts cannot be derived without the initial
```
Comment (profradha) [inline docs/learn/learn-accounts.md]: by "seed", do you mean the “mnemonic phrase" displayed above?

File: docs/learn/learn-accounts.md [hunk 2]
```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
...
+password. In fact, if one knows the seed and the soft or hard derivation path, will have access to
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
password. In fact, if one knows the seed and the soft or hard derivation path, they will have access to
```

File: docs/learn/learn-accounts.md [hunk 3]
```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
...
+your account because the account password is only used to sign transactions on a specific device.
```
Comment (profradha) [inline docs/learn/learn-accounts.md]: That line is confusing - you are referring to the account password set on the extension or the device etc, which can be mistaken for the password in the password derived accounts. Might want to rephrase or remove completely. It is explained the above section that knowing the mnemonic phrase and the derviation path will give full access to the derived account. 

```suggestion
your account.
```

File: docs/learn/learn-accounts.md [hunk 4]
```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
...
+For password-derived accounts, the password is applied on the derivation path meaning that you can
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
For password-derived accounts, the password is applied on the derivation path. You can
```

File: docs/learn/learn-accounts.md [hunk 5]
```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
...
+know the seed phrase and the derivation path, but without the password it is not possible to access
+the account. In mathematical terms, if we have a `written derivation path` and a `password` we can
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
the account. In mathematical terms, if we have a `written derivation path` and a `password`, we can
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: docs/learn/learn-accounts.md (hunk 1 above)
Comment (filippoweb3) [reply docs/learn/learn-accounts.md]: Are seed phrase and mnemonic phrase the same?

Comment (profradha) [reply docs/learn/learn-accounts.md]: Yeah, I think so.
You refer to the string of words as mnemonic phrase above and refer to it as seed below. So, wanted to ensure consistency.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**